backend/schemas/scripts/generate_lookup_schemas.py

```diff
@@ -14,36 +14,64 @@ def lmap(fun, ls):
     return list(map(fun, ls))
 
 
+def process_cfda_lookup(arg):
+    df = pd.read_csv(sys.argv[1], converters={"CFDAEXT": str})
+    # Build a couple of Python objects to render as
+    # JSON, and then as Jsonnet
+    program_names = list(df["FEDERALPROGRAMNAME"])
+
+    unique_prefixes_dict = {}
+    for prefix in df["CFDAPREFIX"]:
+        unique_prefixes_dict[prefix] = prefix
+    unique_prefix_list = list(unique_prefixes_dict.keys())
+
+    unique_cfda_dict = {}
+    for index, row in df.iterrows():
+        unique_cfda_dict[f"{row['CFDAPREFIX']}.{row['CFDAEXT']}"] = 1
+    unique_cfda_list = list(unique_cfda_dict.keys())
+
+    # print(program_names.values[0:10])
+    # print(unique_cfda_list[0:10])
+
```

Comment:
```suggestion
```

Did you want to keep these print statements in here?

Reply:
Nope